kuma/javascript/src/banners.jsx

```diff
@@ -114,43 +114,51 @@ export type BannerProps = {
     // name of the waffle flag that controls the banner, and is also
     // used as part of a localStorage key.
     id: string,
+    // class name used on main banner container. Exclusively used
+    // for styling purposes.
+    classname: string,
     // The banner title. e.g. "MDN Survey"
     title: string,
     // The banner description. e.g. "Help us understand the top 10 needs..."
-    copy: string,
+    // Could also be a React Element such as that returned by `<Interpolated />`
+    copy: Object | string,
```

Comment:
```suggestion
    copy: React.ReactNode | string,
```
I think that should also work and narrows the type down a bit more.

Reply:
Oh I think my mind was in TypeScript mode. It might be `React.Node` in Flow